pyproject.toml

```diff
@@ -1,3 +1,34 @@
+[project]
+name = "torch"
+version = "2.8.0a0"
+description = "Tensors and Dynamic neural networks in Python with strong GPU acceleration"
+readme = "README.md"
+requires-python = ">=3.9"
+license = {text = "BSD"}
+dependencies = [
```

Comment:
This adds another place to maintain the dependencies and make sure they all stay in sync...